cloud-function/src/handlers/proxy-kevel.ts

```diff
@@ -31,7 +29,7 @@ export async function proxyKevel(req: Request, res: Response) {
 
   const userAgent = req.headers["user-agent"] ?? "";
 
-  const parsedUrl = url.parse(req.url);
+  const parsedUrl = new URL(req.url);
```

Comment:
I'm not sure if this would work (document: <https://nodejs.org/api/http.html#http_message_url>):

```suggestion
  const parsedUrl = new URL(req.url, `http://${req.getHeaders().host}`);
```